src/content/about.md

```diff
@@ -0,0 +1,239 @@
+
+The `merge()` plugin will copy the contents of each of its inputs into its output directory. Thus, it merges our 
+uglified JavaScript and out vendor prefixed css. This then becomes our final output and is what is written to 
+our target (destination) directory.
+
+This should all be fairly familiar to you if you've ever written JavaScript (or any programming language for that
+matter) before, it's just inputs and output.
+
+### Plugins
+
+Plugins are what a build pipeline developer will interact with most. Plugins are what do the actual work of
+transforming files at each step of build process. The API of a plugin is pretty simple, all that's required is
+creating a class that extends the [broccoli-plugin](https://github.com/broccolijs/broccoli-plugin) base class, and
+implementing a `build()` method, that performs some work or returns a promise.
+
+```js
+const Plugin = require('broccoli-plugin');
+
+class MyPlugin extends Plugin
+{
+  build() {
+    // A plugin can receive single or multiple inputs nodes/directories
+    // The files are available in this.inputPaths[0]/this.inputPaths[1]...
+    // You can do anything with these files that you can do in node
+    // The output of your plugin must write to this.outputPath
+  }
+}
+```
+
+A broccoli-plugin has only one purpose to transform the files from `this.inputPaths` directories to their output
+directory in this.outputPath directory when its `build()` function is invoked. Anything you can do in node, 
+you can do in the `build()` method. A plugin can receive one or multiple inputs, and these are available in the 
+`this.inputPaths` array in the order they are provided. `this.inputPaths` contains paths to directories, that are
+the `outputPath` of previous plugins. Each `inputPath` contains files that you can manipulate and write to
+`this.outputPath`. Broccoli will handle the state of these directories and passing them between plugins.
+
+There is a special case where a `string` is passed as an input to a plugin. When parsing your build pipeline, 
+Broccoli will automatically convert a string input into a
+[source plugin](https://github.com/broccolijs/broccoli-source). This plugin basically connects its input directory
+directly to its output directory, and also allows Broccoli to `watch` and be notified when files within the input
+directory change and trigger a rebuild. You can also manually create an `unwatched` directory from a string by
+using [UnwatchedDir](https://github.com/broccolijs/broccoli-source#new-unwatcheddirdirectorypath-options).
+
+### Trees
+
+Working from the target directory (the final output) back up to the source directories resembles a tree. Think 
+of a piece of Broccoli and you should have a mental model of what a build pipeline looks like.
+
+Broccoli constructs this tree of connected nodes in memory as it parses the `Brocfile`, and sets up the filesystem 
```

Comment:
Having read this for the first time, I don't know what a `Brocfile` is by this point.  Maybe call if a configuration file, since you will talk about Brocfile again in ##Building section below.